### DISCUSSION TITLE
how to get the size of mesh?

### DISCUSSION THREAD

**Discussion by wild-fruits** [bevyengine/bevy#13619] [Q&A]
how to get the size of mesh?

**Comment by GitGhillie** [comment]
You can query for the [Aabb](https://docs.rs/bevy/latest/bevy/render/primitives/struct.Aabb.html) from which you can get the size of the bounding box

**Reply by wild-fruits** [reply]
Thank you sir for your answer, it helps me.

**Reply by GitGhillie** [reply]
Glad I could help :)
